dependent/addiction-science-and-clinical-practice.csl

```diff
@@ -1,18 +1,16 @@
 <?xml version="1.0" encoding="utf-8"?>
 <style xmlns="http://purl.org/net/xbiblio/csl" version="1.0" default-locale="en-US">
-  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/bmc -->
+  <!-- Generated with https://github.com/citation-style-language/utilities/tree/master/generate_dependent_styles/data/springer -->
   <info>
     <title>Addiction Science &amp; Clinical Practice</title>
-    <title-short>Addict Sci Clin Pract</title-short>
     <id>http://www.zotero.org/styles/addiction-science-and-clinical-practice</id>
     <link href="http://www.zotero.org/styles/addiction-science-and-clinical-practice" rel="self"/>
-    <link href="http://www.zotero.org/styles/biomed-central" rel="independent-parent"/>
-    <link href="http://www.ascpjournal.org" rel="documentation"/>
+    <link href="http://www.zotero.org/styles/springer-basic-brackets" rel="independent-parent"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/Key_Style_Points_1.0.pdf" rel="documentation"/>
+    <link href="http://www.springer.com/cda/content/document/cda_downloaddocument/manuscript-guidelines-1.0.pdf" rel="documentation"/>
     <category citation-format="numeric"/>
-    <category field="biology"/>
-    <category field="medicine"/>
```

Comment:
Because all BMC journals are annotated with the "biology" and "medicine" fields via the BMC template: https://github.com/citation-style-language/utilities/blob/master/generate_dependent_styles/data/bmc/_template.csl